hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbTest.cpp

```diff
@@ -70,34 +70,42 @@ class ServiceHootApiDbTest : public CppUnit::TestFixture
 
 public:
 
-  static QString userEmail() { return "ServiceHootApiDbTest@hoottestcpp.org"; }
+  QString userEmail() { return QString("%1.ServiceHootApiDbTest@hoottestcpp.org").arg(testName); }
+  QString userName()  { return QString("%1.ServiceHootApiDbTest").arg(testName); }
 
   long mapId;
+  QString testName;
 
-  void deleteUser(QString email)
```

Comment:
There already exists a `deleteUser` function in `ServicesDbTestUtils` so this one got deleted.